config/routes.rb

```diff
@@ -17,5 +17,15 @@
   delete "/posts/:id/comments/:id", to: "comments#destroy", as: "delete_comment"
   post "/posts/:id/like", to: "likes#create", as: "user_post_likes"
   get "/posts/:id", to: "posts#show", as: "user_post"
+
+  namespace :api, defaults: {format: 'json'} do
+    namespace :v0 do
+      resources :users, only: :show do
```

Comment:
- [ ] Kindly make sure there are routes for user authentication in for the API or remove authentication altogether from your API